lcls_tools/common/devices/wire.py

```diff
@@ -0,0 +1,405 @@
+
+        def decorated(self, *args, **kwargs):
+            # TODO: Find out right check for this spot/wires
+            if self.motr != "Ready":
+                print("Unable to perform action, wire not in Ready state")
+                return
+            return f(self, *args, **kwargs)
+
+        return decorated
+
+    def check_options(options_to_check: Union[str, List]):
+        """Decorator to only allow motor to move if wire is in the retracted position."""
+        # Maybe this comment should be updated? Would we move from non retracted position?
+        def decorator(function):
+            @wraps(function)
+            def decorated(self, *args, **kwargs):
+                # convert single value to list.
+                if isinstance(options_to_check, str):
+                    options = [options_to_check]
+                for option in options:
+                    if option not in self.controls_information.ctrl_options:
+                        print(
+                            f"unable to perform process {option} with this wire {self.name}"
+                        )
+                        return
+                return function(self, *args, **kwargs)
+
+            return decorated
+
+        return decorator
+
+    @property
+    def xsize(self):
+        """Returns the x wire thickness in um."""
+        # Try to grab from PV first, then if fails, get from yaml
+        # Make sure to print statement saying if yaml values used
+        try:
+            return self.metadata.PVs.xsize.get()
+        except Exception:
+            print(EPICS_ERROR_MESSAGE)
+            # TODO: Returning wire size from yaml file instead
+            return
+
+    @property
+    def ysize(self):
+        """Returns the y wire thickness in um."""
+        try:
+            return self.metadata.PVs.ysize.get()
+        except Exception:
+            print(EPICS_ERROR_MESSAGE)
+            # TODO: Returning wire size from yaml file instead
+
+    @property
+    def usize(self):
+        """Returns the u wire thickness in um."""
+        try:
+            return self.metadata.PVs.usize.get()
+        except Exception:
+            print(EPICS_ERROR_MESSAGE)
+            # TODO: Returning wire size from yaml file instead
+
+    @property
+    def use_x_wire(self):
+        """Checks if the X plane will be scanned."""
+        return self.controls_information.PVs.usexwire.get()
+
+    @use_x_wire.setter
+    def use_x_wire(self, val: bool) -> None:
+        try:
+            BooleanModel(value=val)
+            int_val = int(val)
+            self.controls_information.PVs.usexwire.put(value=int_val)
+        except ValidationError as e:
+            print("Value must be a bool:", e)
+
+    @property
+    def x_range(self):
+        """
+        Returns the X plane scan range.
+        Sets both inner and outer points.
+        """
+        x_inner = self.x_wire_inner
+        x_outer = self.x_wire_outer
+        x_range = [x_inner, x_outer]
```

Comment:
Could we simplify this even more by returning the range directly? 
`return [self.x_wire_inner, self.x_wire_outer]`

Or may we need to do more manipulation here?